generators/spa/templates/VueJS/common/Services/SettingsController.cs

```diff
@@ -0,0 +1,84 @@
+﻿/*
+Copyright Upendo Ventures, LLC 
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), 
+to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+DEALINGS IN THE SOFTWARE.
+*/
+
+using System.Net.Http;
+using System.Net;
+using System.Web.Http;
+using DotNetNuke.Entities.Modules;
+using DotNetNuke.Web.Api;
+using DotNetNuke.Security;
+using Telerik.Web.UI.Calendar.Utils;
+using <%= fullNamespace %>;
+using <%= fullNamespace %>.Services.ViewModels;
+
+namespace <%= fullNamespace %>.Services
+{
+    [SupportedModules("<%= moduleName %>")]
+    [DnnModuleAuthorize(AccessLevel = SecurityAccessLevel.Edit)]
+    public class SettingsController : DnnApiController
+    {
+        public SettingsController() { }
+
+        [HttpGet]  //[baseURL]/settings/load
+        
+        public HttpResponseMessage LoadSettings()
+        {
+            var settings = new SettingsViewModel();
+           
+            if (ActiveModule.ModuleSettings.ContainsKey(QuickSettings.MODSETTING_Name))
```

Comment:
@VAlmea I can't find an example of this right now, but there's a newer way to do this using a single line of code.  Please look for a way to get a specific DNN module setting by name, including a default value/type.  I believe it's in the base class for the module, named `GetModuleSetting` and `GetTabModuleSetting`, respectively.  It might also be coming from the `ModuleController` class.

Reply:
@VAlmea ???

Reply:
Why instantiate the ModuleController class when we have access to the same property inherited from DnnApiController? For this specific case, I only validate that the necessary parameters exist to build the response DTO